.github/workflows/rust.yml

```diff
@@ -165,6 +165,13 @@ jobs:
         run: |
           forest-cli snapshot export
           sha256sum -c *.sha256sum
+      - name: validate snapshot
+        run: |
+          set +e
```

Comment:
You don't need those anymore if I get them right

Reply:
Yep, removed